pkg/arm/v5/resources.go

```diff
@@ -213,14 +217,85 @@ func (g *simpleGenerator) lbKubernetes() *network.LoadBalancer {
 							)),
 						},
 					},
-					Name: to.StringPtr(lbKubernetesOutboundFrontendConfigurationName),
 				},
 			},
 			BackendAddressPools: &[]network.BackendAddressPool{
 				{
 					Name: to.StringPtr(lbKubernetesBackendPoolName),
 				},
 			},
+			LoadBalancingRules: &[]network.LoadBalancingRule{
+				{
+					LoadBalancingRulePropertiesFormat: &network.LoadBalancingRulePropertiesFormat{
+						FrontendIPConfiguration: &network.SubResource{
+							ID: to.StringPtr(resourceid.ResourceID(
+								g.cs.Properties.AzProfile.SubscriptionID,
+								g.cs.Properties.AzProfile.ResourceGroup,
+								"Microsoft.Network/loadBalancers",
+								lbKubernetesName,
+							) + "/frontendIPConfigurations/" + lbKubernetesOutboundFrontendConfigurationName),
```

Comment:
you need to create a new IP, and a new frontend configuration.  Don't reuse outbound.

Reply:
Created new ip and frontend.